src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -10,6 +10,19 @@ import { ResourceType, StringMap, Trigger } from "types/triggers";
 import { toSentenceCase } from "utils/string";
 import { SubscriptionSelect } from "./NotificationsRow";
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const generateWebhookSecret = () => {
```

Comment:
Given Arjun's comment, maybe someone on app should also review?

Also, do you want to hook this function up to the form, or is the goal of this ticket just to make the function available?

Reply:
Hmm I guess I'm not sure what the use of the webhook secret is. Do users need to copy this to use elsewhere? I think it would be okay to push improving this off to a new ticket anyway, since no one has a problem with how it is now, and we could investigate maybe just hiding it completely.

Reply:
sounds good. https://jira.mongodb.org/browse/EVG-17253